src/internal.c

```diff
@@ -9170,6 +9193,7 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                             ret = DOMAIN_NAME_MISMATCH;
```

Comment:
Let's put a MSG here about common name failure, otherwise looks good.

Reply:
If the domain name isn't the common name, line 9164 (left) will log the common name failure, and if the domain name isn't in the subject alt names, you get an additional message on 9168 (left).

Reply:
Ok, thanks, wasn't showing up on the right side.